openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/fabric8/F8OpenShiftAdapter.java

```diff
@@ -473,17 +474,16 @@ protected OpenShiftResourceHandle createResourceFromStream(InputStream stream) t
             String content = IOUtils.toString(stream, StandardCharsets.UTF_8);
 
             try {
-                // TODO only works if JSON should work with YAML as well.
-                ModelNode json;
-                json = ModelNode.fromJSONString(content);
+                final ModelNode json = ModelNode.fromJSONString(content);
```

Comment:
This try/catch block can be replaced by `client.load(stream).createOrReplace()`. Where client is an instance of the kubernetes-client.

Potentiall allowing us to get rid of snakeyml, dmr and maybe commons-io?

Reply:
@iocanel I have updated the code with your suggestion. I have run tests and it works, but can you review?